janus.c

```diff
@@ -309,9 +309,17 @@ static json_t *janus_info(const char *transaction) {
 	json_object_set_new(info, "candidates-timeout", json_integer(candidates_timeout));
 	json_object_set_new(info, "server-name", json_string(server_name ? server_name : JANUS_SERVER_NAME));
 	json_object_set_new(info, "local-ip", json_string(local_ip));
-	if(janus_get_public_ip_count() > 0) {
+	guint public_ip_count = janus_get_public_ip_count();
+	if(public_ip_count > 0) {
 		json_object_set_new(info, "public-ip", json_string(janus_get_public_ip(0)));
 	}
+	if(public_ip_count > 1) {
+		json_t *ips = json_array();
+		for (guint i = 0; i < public_ip_count; i++) {
```

Comment:
Please move the definition of `i` outside of the `for`. I think some compilers barf on that.

Reply:
Done